apps/shared.hu.i18n

```diff
@@ -3,86 +3,86 @@ ActivateExamMode = "A vizsgálati mód aktiválása"
 ActivateDutchExamMode = "A holland vizsga mód aktiválása"
 ActiveExamModeMessage1 = "Az összes adatod"
 ActiveExamModeMessage2 = "törölve lesz ha"
-ActiveExamModeMessage3 = "aktiválod a vizsga módot."
-ActiveDutchExamModeMessage1 = "Az összes adat törlödik, amikor"
-ActiveDutchExamModeMessage2 = "aktiválod a vizsga módot. A Python "
-ActiveDutchExamModeMessage3 = "alkalmazás nem lesz elérhetö."
-Axis = "Tengely"
+ActiveExamModeMessage3 = "a vizsga módot aktiválja."
+ActiveDutchExamModeMessage1 = "Az összes adatod törölve lesz"
+ActiveDutchExamModeMessage2 = "ha a vizsga módot aktiválja. A"
+ActiveDutchExamModeMessage3 = "Python alkalmazás használhatatlan lesz."
+Axis = "Tengelyek"
 Cancel = "Mégse"
 ClearColumn = "Oszlop törlése"
 ColumnOptions = "Oszlop opciók"
-CopyColumnInList = "Az oszlop exportálása egy listába"
-DataNotSuitable = "Az adatok nem megfelelöek"
-DataTab = "Data"
-DefaultSetting = "Alapbeállítások"
+CopyColumnInList = "Az oszlopot egy listába másolni"
+DataNotSuitable = "Az adatok nem felelnek meg"
+DataTab = "Adatok"
+DefaultSetting = "Alapállítások"
 Deg = "deg"
 Deviation = "Varianca"
 DisplayValues = "Értékek mutatása"
 Empty = "Üres"
 Eng = "eng"
-ExitExamMode1 = "Kilépés a vizsga "
+ExitExamMode1 = "Kilépni a vizsga "
 ExitExamMode2 = "módból?"
 Exponential = "Exponenciális"
 FillWithFormula = "Töltse ki egy képlettel"
 ForbiddenValue = "Tiltott érték"
-FunctionColumn = "0 (0) oszlop"
+FunctionColumn = "0(0) oszlop"
 FunctionOptions = "Funkció opciók"
-Goto = "Menj ide"
+Goto = "Menj ..."
 GraphTab = "Grafikon"
-HardwareTestLaunch1 = "Ön elindítja a hardvert"
-HardwareTestLaunch2 = " tesztet.  A teszt végén neked"
-HardwareTestLaunch3 = "kell vissza állítani az eszközt,és"
-HardwareTestLaunch4 = "az összes adat törlödni fog."
+HardwareTestLaunch1 = "A hardverteszt indítása :"
+HardwareTestLaunch2 = "Nyomjon a reset gombra a"
+HardwareTestLaunch3 = "teszt megállításához (ez"
+HardwareTestLaunch4 = "az adatokat törölni fogja)"
 Initialization = "Elöbeállítás"
 IntervalSet = "Állítsa be az intervallumot"
 Language = "Nyelv"
-LowBattery = "Alacsony töltöttség"
+LowBattery = "Majdnem kimerült az elem"

```

Comment:
"Alacsony töltöttség" is more professional and shorter, I think we should go with that.